projects/gnomad/src/client/DownloadsPage.js

```diff
@@ -39,202 +39,202 @@ const DownloadSection = styled.div`
   ],
   [
-    '14.74 GiB',
-    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr14.vcf.bgz',
+    '14.63 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.14.vcf.bgz',
   ],
   [
-    '13.87 GiB',
-    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr15.vcf.bgz',
+    '13.77 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.15.vcf.bgz',
   ],
   [
-    '15.54 GiB',
-    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr16.vcf.bgz',
+    '15.43 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.16.vcf.bgz',
   ],
   [
-    '13.62 GiB',
-    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr17.vcf.bgz',
+    '13.52 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.17.vcf.bgz',
   ],
   [
-    '12.16 GiB',
-    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr18.vcf.bgz',
+    '12.07 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.18.vcf.bgz',
   ],
   [
-    '11.3 GiB',
-    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr19.vcf.bgz',
+    '11.22 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.19.vcf.bgz',
   ],
   [
-    '9.69 GiB',
-    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr20.vcf.bgz',
+    '9.62 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.20.vcf.bgz',
   ],
   [
-    '6.17 GiB',
-    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr21.vcf.bgz',
+    '6.12 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.21.vcf.bgz',
   ],
   [
-    '6.54 GiB',
-    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr22.vcf.bgz',
+    '6.49 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.22.vcf.bgz',
   ],
   [
-    '17.92 GiB',
-    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chrX.vcf.bgz',
+    '17.8 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.X.vcf.bgz',
   ],
 ]
 
 const createVcfLink = file => {
   const fileSize = file[0]
   const url = file[1]
-  const chrom = /chr(\w+)/.exec(url)[0]
+  const chrom = /sites.(\w+)/.exec(url)[0]
```

Comment:
Since the only difference between VCF URLs is the chromosome number, it would be a lot simpler if the list contained only the chromosome instead of the full URL and this function built the URL from the chromosome instead of parsing the chromosome out of the URL.